src/main/io/displayport_msp_bf_compat.c

```diff
@@ -165,11 +167,11 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
 */
     case SYM_WATT:
         return BF_SYM_WATT;
-
+/*
     case SYM_MW:
-        return '^';     // Power symbol for math
```

Comment:
Long shot. I think we are trying too hard.

Reply:
Agree it's a long shot. But do you have a better suggestion other than "?" ?